scripts/jenkins/Jenkinsfile

```diff
@@ -902,6 +923,49 @@ def buildHomepage() {
   return homepageTasks
 }
 
+def deployWebUI() {
```

Comment:
i had hoped we could reuse some of the code in deployHomepage instead of just having a 1:1 copy with some names exchanged.

Reply:
I will try to refactor it a bit